src/action/bandaid.rs

```diff
@@ -1,101 +1,43 @@
 //! A mistake bandaid.
 //!
-//! Covers the mistake with a suggested replacement, as
-//! picked by the user.
+//! A `BandAid` covers the mistake with a suggested
+//! replacement, as picked by the user. It only refers
+//! to suggestions on one line.
+//! Multi-line suggestions are collected in a `FirstAidKit`.
```

Comment:
`FirstAidKits` haven't survived this journey:
```suggestion
//! replacement, as picked by the user.
```